src/user-research/choosing-and-using-analysis-and-synthesis-methods.md

```diff
@@ -40,6 +48,18 @@ Whatever approach we take, our analysis and synthesis goes through three broad s
 
    Working with our colleagues to agree what to do in response to the findings, including adapting our research activities, creating new design ideas, or recording issues to resolve.
 
+### Rich data and big data analysis
+
+Each research activity we do, typically produces one kind of data - notes or transcripts from interviews, videos from usability tests, completed boards or worksheets from workshops, responses from questionnaires.
```

Comment:
@johnwaterworth here you say "one kind of data" but I see multiple kinds of data listed, transcripts are different type of data to videos, and these are different to boards from workshops and so different to questionnaire answers. What am I missing here, what do you mean by one kind of data?

Reply:
What is mean is that each individual research activity typically produces one kind of data.
So from interviews we usually get notes or transcripts.
From usability tests we usually get screen recordings.
From workshops we get completed boards or worksheets.
Etc.

Reply:
@johnwaterworth but in real life, practicality, one research activity will not produce one kind of data in all cases, for example, from a usability test i can get a video, a transcript, and also stickies in a miro/lucid board because i have mixed activities within the usability session, also in usability sessions is typical to have a semi-strutured interview at the start before doing usability testing of specific things. I would like to look at the bigger picture and ask, is this paragraph needed here at all? We as user researchers already now what type of date we can get from research activities and the purpose of this section is to explain what is rich and what is big data and how it relates to the work we do at dxw.  What do you think?